pkg/data/task_result.go

```diff
@@ -35,8 +35,18 @@ type TranscodeTaskOutput struct {
 	Asset UploadTaskOutput `json:"asset,omitempty"`
 }
 
+type TranscodeFileTaskOutputMp4 struct {
+	Renditions []TranscodeFileTaskOutputPath `json:"renditions,omitempty"`
+}
```

Comment:
Do we need this indirection? WDYT of just having the `Mp4` field as a `[]TranscodeFileTaskOutputPath`?

Reply:
I think it's fine. Then, the output will look like this (which I think is ok):
```json
"output": {
    "transcodeFile": {
        "url": "ipfs://bafybeifkjw5443v4wsp6q4cjuzboi7nt7i6n6iinyj7ii6fapqsuavg2hy",
        "hls": {
            "path": "video/index.m3u8"   
        },
        "mp4": [
             { "path": "video/static360p0.mp4" },
             { "path": "video/static720p0.mp4" },
             { "path": "video/static1440p0.mp4" }
         ]         
    }
}
```